Packs/QuestKace/Integrations/QuestKace/QuestKace.py

```diff
@@ -0,0 +1,785 @@
+    """
+    limit = int(args.get('limit', 50))
+    custom_shaping = args.get("custom_shaping")
+    custom_filter = args.get("custom_filter")
+    response = client.tickets_list_request(custom_shaping, custom_filter)
+    raw_response = response.get('Tickets')[:limit]
+    context = parse_response(raw_response)
+    context = add_IsDeleted_key(context)
+    human_readable_markdown = tableToMarkdown(f'Quest Kace Tickets', context, removeNull=True,
+                                              headers=['ID', 'Title', 'Created', 'Modified', 'HdQueueID', 'DueDate',
+                                                       'IsDeleted'])
+    context = {
+        'QuestKace.Ticket(val.ID === obj.ID)': context
+    }
+    return human_readable_markdown, context, raw_response
+
+
+def create_ticket_command(client, args) -> Tuple[str, dict, dict]:
+    """Function which creates a new ticket to the system according to users arguments.
+        Args:
+            client : Integretion client which communicates with the api.
+            args: Users arguments of the command.
+       Returns:
+           human readable, context, raw response of this command.
+    """
+    hd_queue_id = args.get('queue_id')
+    custom_fields = args.get('custom_fields')
+    if not hd_queue_id and not custom_fields:
+        raise DemistoException("queue id is a mandatory value, please add it to your request.")
+    if custom_fields and "hd_queue_id" not in custom_fields:
+        raise DemistoException("hd_queue_id is a mandatory value, please add it to your custom request.")
+    title = args.get("title")
+    summary = args.get('summary')
+    impact = args.get('impact')
+    category = args.get('category')
+    status = args.get('status')
+    priority = args.get('priority')
+    machine = args.get('machine')
+    asset = args.get('asset')
+    # owner = "for now" #??????
+    if not custom_fields:
+        temp_data = create_body_from_args(hd_queue_id, title, summary, impact, category, status, priority, machine,
+                                          asset)
+        temp_data = {'Tickets': [temp_data]}
+    else:
+        temp_data = {'Tickets': [json.loads(custom_fields)]}
+    data = json.dumps(temp_data)
+    demisto.log(data)
+    response = client.create_ticket_request(data)
+    if response.get('Result') == 'Success':
+        try:
+            id = response.get('IDs')[0]
+        except Exception as e:
+            raise DemistoException(e)
+        client.update_token()
+        res = client.ticket_by_id_request(id)
+        ticket = res.get('Tickets')
+        ticket_view = tableToMarkdown(f'New ticket was added successfully, ticket number {id}.\n', ticket)
+        return ticket_view, {}, {}
+    elif response.get('errorDescription'):
+        raise DemistoException(f"Error while adding a new ticket.\n {response.get('errorDescription')}")
+    else:
+        raise DemistoException(f'Error while adding a new ticket.')
```

Comment:
please reverse the order. first check for != `success` and raise, afterwards continue the func logic.